### ISSUE TITLE
consistent version numbers 5.2.2.0 != 5.2.2

### ISSUE THREAD

**Issue by jonasstein** [liballeg/allegro5#780]
the version numbers are not consistent. You would help distributions by using proper version numbers. 
http://semver.org/ is one example. Thank you.
see also https://bugs.gentoo.org/show_bug.cgi?id=623798

**Edit:**
![selection_014](https://user-images.githubusercontent.com/298682/27843392-5afe0030-6113-11e7-9fa3-3b92fbe3952e.png)

Technically there are tools to explain to the gentoo package manager that version 5.2.2.0 should fetch 5.2.2.tar.gz but we prefer to use a variable for the version number and I guess the allegro developers would prefer 5.2.2.0.tar.gz over 5.2.2.tar.gz too.

**Comment by beoran** [comment]
I may be out of line here as a part time Allegro developer, I think that
semver is extremely simplistic and not useful for a project like Allegro.
Rather than semver, we keep almost all of the the non experimental API and
ABI stable and backwards compatible. We have a good track record you can
see here:

https://abi-laboratory.pro/tracker/timeline/allegro/

Furthermore our version numbers have changed  since 5.2 from a three number
to a four number system because we tend to have a lot of bugfix releases
recently. So there never was a 5.2.2 release.

In other words, even though I read the Gentoo bug report, I fail to see how
this is our problem.

Kind Regards,

B.

**Comment by SiegeLord** [comment]
The issue is that the true version is '5.2.2.0', but the file is named '5.2.2', as the '.0' has been customarily dropped. I don't feel to strongly about this, doesn't seem like a terrible thing to just name the file '5.2.2.0' (in fact, that's what it is named by our release making tool, it's just manually renamed to drop the last .0).

**Comment by goobliata** [comment]
No harm in keeping the .0 in the filename.

**Comment by SiegeLord** [comment]
I also updated the 5.2.2.0 release to have the 5.2.2.0 binaries.

**Comment by jonasstein** [comment]
Thank you.
